.babelrc

```diff
@@ -1,11 +1,9 @@
 {
   "presets": [
-    ["es2015", { "modules" : false }],
-    "react",
-    "stage-1"
-  ],
-  "plugins": [
-    "transform-runtime"
+    "transform-runtime",
+    "es2015",
+    "stage-0",
```

Comment:
Some more support for future ES features. Stage 0 includes stage 1. https://babeljs.io/docs/plugins/preset-stage-0/